uint8 my_counter
float64 pos_x
float64 pos_y
float64 pos_theta
string status
